### PR TITLE
Fix bug in drydep_mod from PR #2709 by moving ENDIF further down in routine DEPVEL

### PR DESCRIPTION
**Pull request by yantosca**
### Name and Institution (Required)

Name: Bob Yantosca
Institution: Harvard + GCST

### Describe the update
This is the companion PR to #2726.  It does the following.

1. Implements the fix for the error in PR #2709 described [here](https://github.com/geoschem/geos-chem/pull/2709#discussion_r1953598096) and [here](https://github.com/geoschem/geos-chem/pull/2709#discussion_r1953598643)
2. Update comments for readability, and to fit within 80 columns
3. Trimmed some trailing whitespace

### Expected changes
This PR should allow GCHP simulations to proceed to completion. 

### Related Github Issue
- #2726
- #2709 
- #2705 

Tagging @lizziel @eamarais

### CODE REVIEW

File: GeosCore/drydep_mod.F90 [hunk 1]
```diff
@@ -1293,8 +1293,8 @@ SUBROUTINE DEPVEL( Input_Opt, State_Chm, State_Diag, State_Grid, &
 
     !***********************************************************************
     !
-    !** If LDEP(K)=F, species does not deposit.
-    !** Deposition is applied only to species with LDEP=T.
+    ! If LDEP(K)=F, species does not deposit.
+    ! Deposition is applied only to species with LDEP=T.
```
Comment (yantosca) [inline GeosCore/drydep_mod.F90:1297]: Comment edit only

**Comment by yantosca** [comment]
All GCHP integration tests now pass:
```console
==============================================================================
GCHP: Execution Test Results

CodeDir       : 7e0310f19 GEOS-Chem update: Merge PR #2709 (Remove GOTOs in drydep_mod.F90)
MAPL          : 9ad63ae Merge PR #37 containing update to vertically flip imports with dimensionless pressure proxy lev coordinates
GMAO_Shared   : 4ddb3ec Merge pull request #2 from geoschem/feature/mapl-upgrade
ESMA_cmake    : ad5deba Added ecbuild as a submodule of ESMA_cmake
gFTL-shared   : 4b82492 Merge branch 'upstream_v1.5.0' into feature/v1.5.0
FMS           : 259759d Merge pull request #3 from geoschem/feature/update_gmao_libs
FVdycoreCubed : af42462 Merge PR #8 (Add PLEadv diagnostic for offline advection in GCHP)
geos-chem     : 19036cf5a Move ENDIF statment in drydep_mod.F90 to where 170 CONTINUE was
HEMCO         : 2423647 Merge PR #279 (Use USTAR from met in DustDEAD extension)
yaFyaml       : 19afe50 Merge branch 'upstream_v1.0.4' into feature/v1.0.4
pFlogger      : 2c4b724 Merge branch 'upstream_v1.9.1' into feature/v1.9.1
Cloud-J       : f8a2b7f Update version number for 8.0.1 release
HETP          : 2a99b24 Merge pull request #2 from geoschem/bugfix/initialize_local_variables

Number of execution tests: 12

Submitted as SLURM job: 3782391
==============================================================================

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%  All execution tests passed!  %%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```
Also all integration tests were zero-diff wr//t 14.6.0-alpha.5.
